src/main/java/org/I0Itec/zkclient/ZkClient.java

```diff
@@ -1047,6 +1051,8 @@ public Object call() throws Exception {
             return true;
         } catch (ZkNoNodeException e) {
             return false;
+        } catch (ZkBadVersionException e) {
```

Comment:
@granthenke Sure, I can remove it. I was returning false initially and moved to throwing the exception, but thought it was a bad idea because I'd be overloading the false value and then moved to throwing the exception. In the process, I thought that it would be good to make it explicit that we care about it. I can remove it, though.